jonason/Tokenizer.cpp

```diff
@@ -0,0 +1,103 @@
+//
+//  Tokenizer.cpp
+//  jonason
+//
+//  Created by Gabriel Lumbi on 2023-03-02.
+//
+
+#include "Tokenizer.hpp"
+
+#include <sstream>
+
+namespace jonason {
+
+Token::Token(Token&& other) {
+    tag = std::move(other.tag);
+    value = std::move(other.value);
+    other.value = nullptr;
+}
+
+Token& Token::operator=(Token&& other) {
+    tag = std::move(other.tag);
+    value = std::move(other.value);
+    other.value = nullptr;
+    return *this;
+}
+
+Token::~Token() {
+    delete[] value;
```

Comment:
Gotta love your raw pointers?
You could at least do `std::unique_ptr<char[]>` if you don't want a `std::string`.